### DISCUSSION TITLE
FlinkCDC 在 MySQL 增量同步到 Doris 时候出现了 Stuck

### DISCUSSION THREAD

**Discussion by gilboom** [apache/doris#25057] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/23697209/3d6ca5f8-e017-4b0e-b872-fbd84a2f2b9a)

![image](https://github.com/apache/doris/assets/23697209/fa10461e-278e-44d1-a312-505482bb04c7)

看日志上的错误栈应该是 Flink 在通过 Stream Load 将数据导入 Doris 并读取 HTTP 响应的时候卡住了，然后最后 TaskManager 退出了

![image](https://github.com/apache/doris/assets/23697209/0336e672-0b5d-4e81-b636-e41e74d90f8f)

**Comment by JNSimba** [comment]
You try to request the commit interface, does it work?

**Reply by gilboom** [reply]
之前是没问题的，突然有段时间就 stuck 了。我是用的这个工具进行同步的。[Flink Doris Connector](https://doris.apache.org/zh-CN/docs/dev/ecosystem/flink-doris-connector?_highlight=flink#%E4%BD%BF%E7%94%A8flinkcdc%E6%8E%A5%E5%85%A5%E5%A4%9A%E8%A1%A8%E6%88%96%E6%95%B4%E5%BA%93%E7%A4%BA%E4%BE%8B)

**Reply by JNSimba** [reply]
Which version of doris?
How many tables are synchronized?

**Reply by gilboom** [reply]
doris-1.2.4-1

同步了 16 个库，每个库 30 张表

**Reply by JNSimba** [reply]
Not sure if be is stuck, you can try upgrading to 2.0.2 and try again
